### DISCUSSION TITLE
Just a pointer...

### DISCUSSION THREAD

**Discussion by Kailee71** [Jip-Hop/jailmaker#69] [General]
Hi @Jip-Hop,
just a quick pointer to a discussion about jailmaker in [/r/truenas](https://www.reddit.com/r/truenas/comments/18hfwcr/plans_for_freebsd_14_support/kdw91xh/?context=8&depth=9) in case you missed it; somewhere halfway down the otherwise not related thread I chat with /u/kmoore134 about integrating jailmaker (or something like it) in a future Scale.
Sounds like we need to find more users 😃 to make it happen.

HTH

Kai.

**Comment by Jip-Hop** [comment]
Thanks for letting me know! Interesting discussion and sounds like good news to me! 😀

**Comment by rowanmoul** [comment]
Kris Moore changed the [Jira ticket](https://ixsystems.atlassian.net/browse/NAS-119787) to "Suggestion Accepted" last week!
